acme/acme/messages.py

```diff
@@ -522,8 +521,6 @@ class Order(ResourceBody):
     :ivar .Error error: Any error that occurred during finalization, if applicable.
     """
     identifiers = jose.Field('identifiers', omitempty=True)
-    status = jose.Field('status', decoder=Status.from_json,
```

Comment:
Although we don't use the returned status, sounds like an `Order` object can come back with a status inside it, which we should retain the ability to parse. So we just want to get the default out I think so that it doesn't fill it in for a `NewOrder`. This is based on my understanding of #5857; if you interpreted that differently or think that we will be able to successfully parse a response that contains a status even without that line, let me know.

Reply:
This is what the initial changes were supposed to handle. Do you feel like populating the `messages.Order` object with the `status` field  when called for parsing JSON payload would be the correct way to take care of this?

Reply:
We should populate it on parse so we can find out the status. We shouldn't be posting it back with a status.